index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head style= "background-color: #e5ba57">
+<head class= "indexHead">
```

Comment:
The `<head></head>` tag in HTML documents is not a visible element. Remove the class.